### PR TITLE
(trivial) Fix a typo in "--help" command line description

### PR DESCRIPTION
**Pull request by rluzynski**
This typo makes the message untranslated.

### CODE REVIEW

**Comment by xsuchy** [comment]
Thank you.

**Comment by ernestask** [comment]
Why do you say it’s a typo?

**Comment by rluzynski** [comment]
> Why do you say it’s a typo?

If you execute ```gnome-abrt --help``` in any non-English language without this commit you will see that ```Select problem ID``` is not translated. If you wanted to reword this to ```Select problem ID``` then feel free to change it but:

1. You should introduce the same change in every occurrence of the string.
2. I suggest you to introduce this change later, maybe after F31 release, to give the translator a chance to translate messages in a stable state.

**Comment by ernestask** [comment]
That does not answer my question, though. It could have been intentional, but you make it appear as though it wasn’t because of a technicality.
